kfdefs/overlays/moc/zero/opf-trino/secrets/opf-datacatalog-bucket.enc.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: ENC[AES256_GCM,data:ieX0WDMVg53P1BqhFiMkT51auc7qRw==,iv:3ibwns9X901tLXVeS3Ly3/kmxfAAV1pwAzhhBCocqms=,tag:PaCSGDQbOioXU9X5nm/d9A==,type:str]
```

Comment:
why is the name encrypted? this doesn't seem to be respecting the sops.yaml that was added